geoproj: add hardcoded set of supported SRIDs

otan opened the pull request:
Add a script that converts a CSV of supportable SRIDs into a hardcoded
SRID map.

Then generate SRIDs based on this criteria:
```
(srid IN (3395, 3574, 3995, 3785, 3857, 3409, 3031, 4004, 4326) OR
(srid >= 32601 and srid <= 32660) OR (SRID >= 32701 AND SRID <= 32760)
OR (proj4text LIKE '%laea%')
OR (srtext ilike '%popular%')
```

These ones are selected because they seem to be used for _ST_BestSRID /
are hardcoded in PostGIS.

Release note: None

cockroach-teamcity commented:
<!-- Reviewable:start -->
This change is [<img src="https://reviewable.io/review_button.png" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/cockroachdb/cockroach/50005)
<!-- Reviewable:end -->

otan committed:
geoproj: add hardcoded set of supported SRIDs

Add a script that converts a CSV of supportable SRIDs into a hardcoded
SRID map.

Then generate SRIDs based on this criteria:
```
(srid IN (3395, 3574, 3995, 3785, 3857, 3409, 3031, 4004, 4326) OR
(srid >= 32601 and srid <= 32660) OR (SRID >= 32701 AND SRID <= 32760)
OR (proj4text LIKE '%laea%')
OR (srtext ilike '%popular%')
```

These ones are selected because they seem to be used for _ST_BestSRID /
are hardcoded in PostGIS.

Release note: None

sumeerbhola reviewed (approved):
<img class="emoji" title=":lgtm:" alt=":lgtm:" align="absmiddle" src="https://reviewable.io/lgtm.png" height="20" width="61"/> 

> Reviewed 12 of 12 files at r1.
> *[Reviewable](https://reviewable.io/reviews/cockroachdb/cockroach/50005#-:-M9VvcrC5C6X7s9yOPqU:bg7igkv)* status: :shipit: complete! 1 of 0 LGTMs obtained



<!-- Sent from Reviewable.io -->

otan commented:
bors r=sumeerbhola
